pkg/trace/pb/ddsketch.proto

```diff
@@ -0,0 +1,66 @@
+/* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License 2.0.
```

Comment:
Can't the DDSketch library do all of this for you? I would expect to be able to call `proto.Unmarshal(bytes, &sketch)` and then call [this](https://github.com/DataDog/sketches-go/blob/master/ddsketch/ddsketch.go#L145) in DDSketch

Reply:
Good point about using the library protobuf.
I updated the code to use their struct. But I need to test the performance on that, because it's doing more allocations, and also, this looks really inefficient:
https://github.com/DataDog/sketches-go/blob/master/ddsketch/store/dense_store.go#L166

Reply:
Indeed. Perhaps we could improve it afterwards though?

Reply:
If we can do it using the DDSketch library as suggested, let's drop this protobuf definition. Performance is not _that_ critical for a request coming in once every 10 seconds here, and I agree with Richard that we can improve on the upstream later on if we must. It's best to share this code for obvious reasons.

Reply:
That makes sense:
I'm blocked on problems with versions of protoc right now, I opened PRs to fix it:
https://github.com/DataDog/sketches-go/pull/19
also the decoding of mapping should be done in the sketch repo: https://github.com/DataDog/sketches-go/pull/20

while waiting on review on these PRs, I need to use this protobuf (but I will update it when my PRs are merged).

Reply:
Actually, the problem was the version of the protobuf library we were using. I changed it and I was able to remove this protobuf and use the one from the sketches repo.